floodviz/templates/google_analytics.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
Sorry about this, I must not have hit add but you should only have the markup in here that you want included. This doesn't need to be valid html document. So you really only need lines 6 through 38